README instructions outdated

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
```
~/git/opa-gatekeeper (master=)$ export DESTINATION_GATEKEEPER_DOCKER_IMAGE=opa/gatekeeper:local
~/git/opa-gatekeeper (master=)$ make docker-build REPOSITORY="opa/gatekeeper:local"
make: *** No rule to make target 'docker-build'.  Stop.
~/git/opa-gatekeeper (master=)$ 
```
If I try to use `make docker-buildx` instead (closest target I could find), it finally dies with:
```
docker: 'buildx' is not a docker command.
...
```


**What did you expect to happen:**
Build to occur

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

Please fix the Makefile or update README

**Environment:**
Ubuntu Linux 18.04

- Gatekeeper version: master (commit f6937bf)
- Kubernetes version: (use `kubectl version`): N/A

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,3 @@
-# Enable Docker CLI experimental features
-DOCKER_CLI_EXPERIMENTAL=enabled
-
 # Image URL to use all building/pushing image targets
 REGISTRY ?= quay.io
 REPOSITORY ?= $(REGISTRY)/open-policy-agent/gatekeeper
@@ -183,35 +180,31 @@ docker-push-release:  docker-tag-release
 	@docker push $(REPOSITORY):$(VERSION)
 	@docker push $(REPOSITORY):latest
 
-# Build the docker image
-docker-build: test
-	docker build --pull . -t ${IMG}
-
 # Build docker image with buildx
 # Experimental docker feature to build cross platform multi-architecture docker images
 # https://docs.docker.com/buildx/working-with-buildx/
 docker-buildx: test
-	@if ! docker buildx ls | grep -q container-builder; then\
-		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder --use;\
+	if ! DOCKER_CLI_EXPERIMENTAL=enabled docker buildx ls | grep -q container-builder; then\
+		DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name container-builder --use;\
 	fi
-	docker buildx build --platform "linux/amd64" \
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/amd64" \
 		-t $(IMG) \
 		. --load
 
 docker-buildx-dev: test
-	@if ! docker buildx ls | grep -q container-builder; then\
-		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder --use;\
+	@if ! DOCKER_CLI_EXPERIMENTAL=enabled docker buildx ls | grep -q container-builder; then\
+		DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name container-builder --use;\
 	fi
-	docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
 		-t $(REPOSITORY):$(DEV_TAG) \
 		-t $(REPOSITORY):dev \
 		. --push
 
 docker-buildx-release: test
-	@if ! docker buildx ls | grep -q container-builder; then\
-		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder --use;\
+	@if ! DOCKER_CLI_EXPERIMENTAL=enabled docker buildx ls | grep -q container-builder; then\
+		DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name container-builder --use;\
 	fi
-	docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
 		-t $(REPOSITORY):$(VERSION) \
 		-t $(REPOSITORY):latest \
 		. --push

--- a/Makefile
+++ b/Makefile
@@ -183,6 +183,13 @@ docker-push-release:  docker-tag-release
 	@docker push $(REPOSITORY):$(VERSION)
 	@docker push $(REPOSITORY):latest
 
+# Build the docker image
+docker-build: test
+	docker build --pull . -t ${IMG}
+
+# Build docker image with buildx
+# Experimental docker feature to build cross platform multi-architecture docker images
+# https://docs.docker.com/buildx/working-with-buildx/
 docker-buildx: test
 	@if ! docker buildx ls | grep -q container-builder; then\
 		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder --use;\
@@ -191,9 +198,6 @@ docker-buildx: test
 		-t $(IMG) \
 		. --load
 
-# Build docker image with buildx
-# Experimental docker feature to build cross platform multi-architecture docker images
-# https://docs.docker.com/buildx/working-with-buildx/
 docker-buildx-dev: test
 	@if ! docker buildx ls | grep -q container-builder; then\
 		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder --use;\

--- a/README.md
+++ b/README.md
@@ -90,6 +90,7 @@ kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/
 Currently the most reliable way of installing Gatekeeper is to build and install from HEAD:
 
    * Make sure that:
+       * You have Docker version 19.03 or later installed.
        * [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder#getting-started) and [Kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md) are installed.
        * Your kubectl context is set to the desired installation cluster.
        * You have a container registry you can write to that is readable by the target cluster.

--- a/README.md
+++ b/README.md
@@ -104,7 +104,7 @@ Currently the most reliable way of installing Gatekeeper is to build and install
       ```
    * Build and push your Docker image:
       ```sh
-      make docker-build REPOSITORY="$DESTINATION_GATEKEEPER_DOCKER_IMAGE"
+      make docker-buildx REPOSITORY="$DESTINATION_GATEKEEPER_DOCKER_IMAGE"
       make docker-push-release REPOSITORY="$DESTINATION_GATEKEEPER_DOCKER_IMAGE"
       ```
    * Finally, deploy:

